pages/how-to-configure.md

```diff
@@ -4,9 +4,9 @@
 
 ## How to Configure
 
-💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`).
+💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`) and you can launch it using your own SCI/SCII variant(s). Alternatively, you may also use the tool by means of defining Profiles as described bellow.
 
-⚠️The below documentation is valid for our own `Profile Management Tool` feature only.
+⚠️The below documentation is valid for our own `Profile Management Tool` (transaction: `Y_CODE_PAL_PROFILE`) feature only.
```

Comment:
The transaction is not duplicated (line 9 and 22). How about if we link to the `Profile Management Tool` page?